modules/setting/setting.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/gogits/gogs/modules/bindata"
 	"github.com/gogits/gogs/modules/log"
 	"github.com/gogits/gogs/modules/user"
+
+	"strk.kbt.io/projects/go/libravatar"
```

Comment:
Are you really going to refuse using anything that's not on github.com ?
Or do you have a list of "whiltelisted" hosting places to pick from ?